Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -1130,6 +1144,8 @@ You can send the message `help` in order to see the supported commands:
     
     Try inserting your configured message endpoint in a browser tap, click `Enter`, if `Method Not Allowed` is returned, the endpoint is valid and ready to communicate, otherwise, it needs to be handled according to the returned error's message.
 
+    In some cases, a connection is not created between Teams and the messaging endpoint, when adding a bot to the team. You can work around this problem by adding any member to the team the bot was added to (the bot should be already added to the team). It will trigger a connection and solve the issue, then you can remove the member that was added with no poroblems.
```

Comment:
```suggestion
    In some cases, a connection is not created between Teams and the messaging endpoint, when adding a bot to the team. You can work around this problem by adding any member to the team the bot was added to (the bot should be already added to the team). This will trigger a connection and solve the issue. You can then remove the member that was added.
```